lib/routes/hdu/auto/notice.ts

```diff
@@ -14,14 +34,35 @@ export const route: Route = {
         supportPodcast: false,
         supportScihub: false,
     },
-    name: '自动化学院 - 通知公告',
+    name: '自动化学院',
     maintainers: ['jalenzz'],
-    handler: () => fetchAutoNews('3779/list.htm', '通知公告'),
-    description: '杭州电子科技大学自动化学院通知公告',
+    handler: (ctx) => {
+        let type = ctx.req.param('type') || 'notice';
+        if (!(type in typeMap)) {
+            logger.error(`Invalid type: ${type}. Valid types are: ${Object.keys(typeMap).join(', ')}, defaulting to notice`);
+            type = 'notice';
+        }
+        return fetchAutoNews(typeMap[type].path, typeMap[type].name);
+    },
+    description: `| 通知公告  | 研究生教育 |    本科教学    | 学生工作  |
+                  | -------- | -------- |   --------    | -------- |
+                  | notice   | graduate | undergraduate | student  |`,
```

Comment:
```suggestion
| -------- | -------- |   --------    | -------- |
| notice   | graduate | undergraduate | student  |`,
```
Indent description with either 0 or 2 spaces, or the markdown content won't be rendered properly.

Reply:
It has been corrected. Thanks.